### DISCUSSION TITLE
FineCover Prototype

### DISCUSSION THREAD

**Discussion by FortuneN** [FortuneN/FineCodeCoverage#212] [General]
@tonyhallett Tell me what you think of this : https://github.com/FortuneN/CoverPrototype

**Comment by tonyhallett** [comment]
I do not have the knowledge to determine if Mono.Cecil is up to the task ( for .Net Core and .Net Framework if you intend to do both ) or if your usage is indeed correct.  ( The results from the tests are approximately the same as per coverlet ). I do not have the time or inclination currently to learn this area.

As for your tests.  If you then perform assertions against the generated file then great ! 

I would question why you would want to take a stab at it though ? I imagine that it will take a substantial amount of time and effort to reach the maturity of OpenCover and Coverlet.


Can you answer my question to you in here please https://github.com/FortuneN/FineCodeCoverage/issues/209

**Reply by FortuneN** [reply]
> As for your tests. If you then perform assertions against the generated file then great !

It's a prototype
The tests are there just to run the code (not to **test** anything)

> I would question why you would want to take a stab at it though

I would want to take a stab at it because i just do so for things that interest me (e.g. creating and publishing a visual studio extension)

> I imagine that it will take a substantial amount of time and effort to reach the maturity of OpenCover and Coverlet.

I imagine so a well but the destination is not the point.
It's the adventure that i want and was hoping you would be interested as well.

Never-the-less, i'll send you post-cards :)
